AutoNode/util.py

```diff
@@ -0,0 +1,57 @@
+from pyhmy import cli
+from pyhmy import (
+    Typgpy,
+    json_load
+)
+
+from .common import (
+    node_config
+)
+
+from .blockchain import (
+    get_block_by_number,
+)
+
+
+def process_passphrase(proc, passphrase, double_take=False):
+    """
+    This will enter the `passphrase` interactively given the pexpect child program, `proc`.
+    """
+    proc.expect("Enter passphrase:\r\n")
+    proc.sendline(passphrase)
+    if double_take:
+        proc.expect("Repeat the passphrase:\r\n")
+        proc.sendline(passphrase)
+        proc.expect("\n")
+
+
+def input_with_print(prompt_str):
+    user_input = input(prompt_str).strip()
+    print(f"{Typgpy.OKBLUE}You entered: `{Typgpy.OKGREEN}{user_input}{Typgpy.OKBLUE}`{Typgpy.ENDC}")
+    return user_input
+
+
+def check_min_bal_on_s0(address, amount, endpoint=node_config['endpoint']):
+    balances = json_load(cli.single_call(f"hmy --node={endpoint} balances {address}"))
```

Comment:
what happens when wrong endpoint used or reply is connection refused, will single_call handle that

Reply:
single_call has a timeout of 60 seconds by default. But added a param for a timeout.